test/Microsoft.Azure.WebJobs.Extensions.EventHubs.Tests/WebJobs.Extensions.EventHubs.Tests.csproj

```diff
@@ -27,6 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs.Host.TestCommon" Version="3.0.10" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="3.0.10" />
```

Comment:
we have 3.0.14 (prerelease) on myget right now that you could reference. https://github.com/Azure/azure-webjobs-sdk/wiki#latest-builds. Before we release, we'd need to make sure we swap this to the official.